pxtcompiler/emitter/service.ts

```diff
@@ -933,8 +933,10 @@ namespace ts.pxtc.service {
                     /^__/.test(si.namespace) || // ignore namespaces starting with _-
                     si.attributes.hidden ||
                     si.attributes.deprecated ||
-                    // don't members with an alias
-                    si.attributes.alias
+                    // don't emit members with an alias
+                    si.attributes.alias ||
```

Comment:
@pelikhan , why do we want to omit members with an alias from completions? This is filtering out entries like "AnimalMob.Chicken" because it has an alias of "CHICKEN". However "CHICKEN" does not appear as its own entry in the apiInfo so CHICKEN is now not appearing at all in completions.

Reply:
Humm CHICKEN did appear in completion. Do you have a context?